spec/spec_helper.rb

```diff
@@ -1,5 +1,24 @@
 require 'simplecov'
 require 'simplecov-console'
+require 'capybara'
+require 'capybara/rspec'
+require 'rspec'
+require_relative './setup_test_database'
+
+ENV['RACK_ENV'] = 'test'
+ENV['ENVIRONMENT'] = 'test'
```

Comment:
consider just using the `RACK_ENV` variable to indicate that you're in the test environment. do you need an extra `ENVIRONMENT` variable too?